package.json

```diff
@@ -41,7 +41,8 @@
     "vue2-filters": "^0.10.0",
     "vuelidate": "^0.7.5",
     "vuetify": "^2.3.9",
-    "vuex": "^3.5.1"
+    "vuex": "^3.5.1",
+    "webpack-remove-serviceworker-plugin": "^1.0.0"
```

Comment:
You can read up on this in the references I put in vue.config.js.

TLDR; You need to register a new Service Worker to replace the one currently on users' machines. But this new SW self-destroys. After some time (a year?) we can delete this package, since all users would have "wiped" their computers by then.

Reply:
New ticket to remove this plugin in 1 year: https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/10370

Reply:
cool!